ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/ChainDownloader.java

```diff
@@ -105,6 +106,10 @@ public ChainDownloader(
                       LOG.trace("Download cancelled", t);
                     } else if (rootCause instanceof InvalidBlockException) {
                       LOG.debug("Invalid block downloaded", t);
+                    } else if (rootCause instanceof PeerDisconnectedException) {
```

Comment:
```suggestion
                    } else if (rootCause instanceof EthTaskException) {
                        LOG.debug(rootCause.getMessage());
                    }
```

Reply:
While you're at it, you could probably just handle all `EthTaskException`'s ^ as those are "expected" exceptions.  It seems that `LOG.error` below should be there to catch unexpected exceptions.

Reply:
Yep.  We don't need stack traces when we fail in expected ways.